src/models/user.d.ts

```diff
@@ -95,6 +95,13 @@ declare namespace rest_api {
             registration_date: "2018-04-09T13:04:44.987830Z";
             vacation_left: number;
             on_vacation: boolean;
+            self_reported_account_linkages?: {
+                org1: string;
+                org1_id: string;
+                org1_rank: string;
+                last_updated: number;
+            };
```

Comment:
Should/could we use https://github.com/online-go/online-go.com/pull/1767/files#diff-3634b4c362041964a17503f2ba9d107297f02376af3788b4648644d880de5887R546-R556 here too? Also FYI there's a `hidden: boolean` as well.

(That link isn't loading initially because User.tsx isn't being rendered by github, but in that file you have a nice explicit list of those that goes something like

```
type Server = "kgs" | "igs" | "dgs" | "golem" | "wbaduk" | "tygem" | "fox" | "yike" | "goquest";

type AccountLinks = {
    [org in `org${1 | 2 | 3}`]?: string;
} & {
    [org_id in `org${1 | 2 | 3}_id`]?: string;
} & {
    [org_rank in `org${1 | 2 | 3}_rank`]?: string;
} & {
    [server_username in `${Server}_username`]?: string;
} & {
    [server_rank in `${Server}_rank`]?: string;
};
```

Reply:
Added! (and `hidden_ids` too!)